src/main/java/org/craftercms/studio/controller/rest/v2/UsersController.java

```diff
@@ -191,7 +191,7 @@ public ResponseBody deleteUser(
      * @param userId User identifier
      * @return Response containing user
      */
-    @GetMapping("/api/2/users/{id}")
+    @GetMapping("/api/2/users/{id:.+}")
```

Comment:
What happens if someone calls the endpoint with an extension? for example `../api/2/users/john.doe.json` will the `id` value include the extension?

Reply:
Jepp, hmmm you're right. That is an issue: 

`
{
    response: {
        code: 6000,
        message: "User not found",
        remedialAction: "Check if you're using the correct User ID",
        documentationUrl: ""
    }
}`

Reply:
@suelje maybe you can try adding this configuration and see if it fixes the issue without breaking the extension recognition: `<mvc:path-matching registered-suffixes-only="true"/>`